app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -270,9 +272,11 @@ public void onConnected(Bundle bundle) {
     }
 
     private void initSummitNotificationHandler() {
-        NotificationHandler notificationHandler = new NotificationHandler(this);
-        if (notificationHandler.shouldSetAlarm()) {
-            notificationHandler.setAlarmForNotification();
+        for (TaggedEventSeries eventSeries : App.getInstance().currentTaggedEventSeries()) {
```

Comment:
`shouldSetAlarm` method below should prevent that from happening. It will set the alarm for only once, right? But this will be called in every Activity creation.
Should we then move this to the `App` class? That way we would do this only once per app start.